iphone/Maps/Bookmarks/Categories/BMCView/BMCViewController.swift

```diff
@@ -105,15 +105,15 @@ final class BMCViewController: MWMViewController {
     let settingsController = CategorySettingsViewController(bookmarkGroup: BookmarksManager.shared().category(withId: category.categoryId))
     settingsController.delegate = self
 
-    MapViewController.topViewController().navigationController?.pushViewController(settingsController,
+    MapViewController.shared()?.navigationController?.pushViewController(settingsController,
                                                                                    animated: true)
```

Comment:
nit: one-liners here and below are more readable...
```suggestion
    MapViewController.shared()?.navigationController?.pushViewController(settingsController, animated: true)
```